src/test/scala/com/atomist/rug/runtime/js/JavaScriptCommandHandlerTest.scala

```diff
@@ -1,111 +1,102 @@
-import com.atomist.rug.runtime.js.interop.jsPathExpressionEngine
+import com.atomist.rug.runtime.plans._
+import com.atomist.rug.spi.Handlers.{InstructionResponse, Status}
+import com.atomist.rug.spi.Secret
 import com.atomist.rug.ts.TypeScriptBuilder
 import com.atomist.source.{SimpleFileBasedArtifactSource, StringFileArtifact}
-import com.atomist.tree.TreeMaterializer
 import org.scalatest.{FlatSpec, Matchers}
 
-class JavaScriptCommandHandlerTest extends FlatSpec with Matchers{
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class JavaScriptCommandHandlerTest extends FlatSpec with Matchers {
 
   val atomistConfig: AtomistConfig = DefaultAtomistConfig
 
   val kitties = "ShowMeTheKitties"
   val kittyDesc = "Search Youtube for kitty videos and post results to slack"
-  val simpleCommandHandler =  StringFileArtifact(atomistConfig.handlersRoot + "/Handler.ts",
-    s"""
-       |import {HandleCommand, Instruction, Response, HandlerContext, Plan, Message} from '@atomist/rug/operations/Handlers'
-       |import {CommandHandler, Parameter, Tags, Intent} from '@atomist/rug/operations/Decorators'
-       |
-       |@CommandHandler("ShowMeTheKitties","Search Youtube for kitty videos and post results to slack")
-       |@Tags("kitty", "youtube", "slack")
-       |@Intent("show me kitties","cats please")
-       |class KittieFetcher implements HandleCommand{
-       |
-       |  @Parameter({description: "his dudeness", pattern: "^.*$$"})
-       |  name: string
-       |
-       |  handle(ctx: HandlerContext) : Plan {
-       |    let pxe = ctx.pathExpressionEngine()
-       |
-       |    if(this.name != "el duderino") {
-       |      throw new Error("This will not stand");
-       |    }
-       |    let result = new Plan()
-       |    result.add({ instruction: {
-       |                 kind: "execute",
-       |                 name: "HTTP",
-       |                 parameters: {method: "GET", url: "http://youtube.com?search=kitty&safe=true", as: "JSON"}
-       |               },
-       |               onSuccess: {kind: "respond", name: "Kitties"},
-       |               onError: {text: "No kitties for you today!"}})
-       |    return result;
-       |  }
-       |}
-       |
-       |export let command = new KittieFetcher();
-       |
+  val simpleCommandHandler = StringFileArtifact(atomistConfig.handlersRoot + "/Handler.ts",
+    """
+      |import {HandleCommand, Instruction, Response, HandlerContext, Plan, Message} from '@atomist/rug/operations/Handlers'
+      |import {CommandHandler, Parameter, Tags, Intent} from '@atomist/rug/operations/Decorators'
+      |
+      |@CommandHandler("ShowMeTheKitties","Search Youtube for kitty videos and post results to slack")
+      |@Tags("kitty", "youtube", "slack")
+      |@Intent("show me kitties","cats please")
+      |class KittieFetcher implements HandleCommand{
+      |
+      |  @Parameter({description: "his dudeness", pattern: "^.*$$"})
```

Comment:
I don't think you need double `$` here since the string in not interpolated (quotes are not prepended with `s`).